guides/common/modules/proc_adding-a-bmc-interface.adoc

```diff
@@ -1,30 +1,45 @@
 [id="Adding_a_Baseboard_Management_Controller_Interface_{context}"]
 = Adding a baseboard management controller (BMC) interface
 
+{Project} can control the power status of bare-metal hosts.
 Use this procedure to configure a baseboard management controller (BMC) interface for a host that supports this feature.
 
 .Prerequisites
-* The `ipmitool` package is installed.
-* You know the MAC address, IP address, and other details of the BMC interface on the host, and the appropriate credentials for that interface.
+* You know the MAC address, IP address, and other details of the BMC interface on the host, and authentication credentials for that interface.
 +
 [NOTE]
 ====
 You only need the MAC address for the BMC interface if the BMC interface is managed, so that it can create a DHCP reservation.
 ====
 
 .Procedure
-. Enable BMC on the {SmartProxy} server if it is not already enabled:
-.. Configure BMC power management on {SmartProxyServer} by running the `{foreman-installer}` script with the following options:
+. Install `ipmitool` on your {SmartProxy}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {foreman-installer} --foreman-proxy-bmc=true \
---foreman-proxy-bmc-default-provider=ipmitool
+# {project-package-install} ipmitool
```

Comment:
This shouldn't be needed. It looks like this is a regression introduced in https://github.com/theforeman/foreman-packaging/commit/6cc3c98edd01fe2f24db8a426b9eed405c283200 so I've opened https://github.com/theforeman/foreman-packaging/pull/10556 instead. Interestingly enough, it looks like it was never present in Debian packages.